test/controllers/project_controller_test.exs

```diff
@@ -1,113 +0,0 @@
-defmodule ApiWeb.ProjectControllerTest do
```

Comment:
How can we know we didn't break the frontend? I'm assuming we have to do a bunch of greps, on something?

Roping @jgradim in.

Reply:
We're definitely breaking the frontend with this PR. This "remove projects" task needs to be done in the frontend as well

Reply:
Ok, let's depend on that change first, then. Since it's just deleting code, I'm pretty ok with jamming progress for a little while.